argus/backend/service/client_service.py

```diff
@@ -14,6 +19,7 @@ class ClientService:
     PLUGINS = {name: plugin.model for name, plugin in AVAILABLE_PLUGINS.items()}
 
     def __init__(self) -> None:
+        self.notification_manager = NotificationManagerService()
```

Comment:
why not to create it in submit_run method if needed only?

Reply:
Could be useful for other methods in the future.